internal/benchrunner/runners/rally/metrics.go

```diff
@@ -287,9 +286,14 @@ func (c *collector) collectDiskUsage() map[string]ingest.DiskUsage {
 }
 
 func (c *collector) collectMetricsAfterRallyRun() {
-	_, err := c.esAPI.Indices.Refresh(c.esAPI.Indices.Refresh.WithIndex(c.datastream))
+	resp, err := c.esAPI.Indices.Refresh(c.esAPI.Indices.Refresh.WithIndex(c.datastream))
 	if err != nil {
-		logger.Errorf("unable to refresh data stream at the end of rally run")
+		logger.Errorf("unable to refresh data stream at the end of rally run: %s", err)
+		return
+	}
+	defer resp.Body.Close()
+	if resp.IsError() {
+		logger.Errorf("unable to refresh data stream at the end of rally run: %s", resp)
```

Comment:
Should it be used `resp.String()` as here ? 
https://github.com/elastic/elastic-package/pull/1549/files#diff-e0b74fb572545d017ff634116e9791993aa6840936640ece5aa5ffa9549de539R348

Reply:
Indeed, good catch :+1: